{
  "loc.friendlyName": "Python twine アップロードの認証",
  "loc.helpMarkDown": "[Learn more about this task](https://docs.microsoft.com/azure/devops/pipelines/tasks/package/twine-authenticate) or [see the Twine documentation](https://twine.readthedocs.io/)",
  "loc.description": "twine を使用して Python ディストリビューションをアップロードするための認証を行います。'-r FeedName/EndpointName --config-file $(PYPIRC_PATH)' を twine アップロード コマンドに追加します。この組織にフィードが存在する場合は、フィード名をリポジトリ (-r) として使用します。それ以外の場合は、サービス接続に定義されているエンドポイント名を使用します。",
  "loc.instanceNameFormat": "Twine 認証 $(message)",
  "loc.group.displayName.feedAuthentication": "フィードと認証",
  "loc.group.displayName.advanced": "詳細設定",
  "loc.input.label.artifactFeed": "My feed (select below)",
  "loc.input.help.artifactFeed": "Select a feed to authenticate present in this organization.",
  "loc.input.label.pythonUploadServiceConnection": "Feed from external organizations",
  "loc.input.help.pythonUploadServiceConnection": "Select an endpoint to authenticate outside this organization. Make sure the endpoint has package upload permissions.",
  "loc.messages.Info_AddingInternalFeeds": "Adding auth information for %s internal feed.",
  "loc.messages.Info_AddingExternalFeeds": "%s 外部エンドポイントの認証情報を追加しています。",
  "loc.messages.Info_AddingAuthForRegistry": "Adding authentication to configuration for registry %s",
  "loc.messages.Info_SuccessAddingAuth": "Successfully added auth for %s internal feed and %s external endpoint.",
  "loc.messages.Info_AddingPasswordAuthEntry": "フィード %s のユーザー名パスワード認証エントリを追加しています",
  "loc.messages.Info_AddingTokenAuthEntry": "フィード %s のトークン認証エントリを追加しています",
  "loc.messages.Error_FailedToParseFeedUrlAndAuth": "フィード URL の解析および認証情報の追加に失敗しました。%s",
  "loc.messages.Warning_OnlyOneFeedAllowed": "Only one internal feed can be authorized at a time using this task. All the feeds except first one will be ignored.",
  "loc.messages.Warning_OnlyOneServiceConnectionAllowed": "Only one service connection can be authorized at a time using this task. All the connections except first one will be ignored.",
  "loc.messages.Error_DuplicateEntryForFeed": "This is a duplicate entry for the feed name. Use unique names to avoid potential publish issues.",
  "loc.messages.FailedToGetPackagingUri": "既定のコレクション URI を使用して、パッケージ URI を取得できません。",
  "loc.messages.FailedToAddAuthentication": "認証を追加できませんでした。",
  "loc.messages.NoPypircFile": "クリーンアップするための .pypirc ファイルがありません。",
  "loc.messages.VariableSetForPypirc": ".pypirc の環境変数が %s で設定されています。",
  "loc.messages.Info_RemovingPypircFile": "%s から .pypirc ファイルを削除しています"
}